### PR TITLE
Updated known issues

### PR DESCRIPTION
**Pull request by nehakulkarni123**
Resolution is available for 2 known issues, added a known issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -278,4 +271,12 @@ This behavior is also seen in resource types from the following RPs:
 - Microsoft.DBforMySQL
 - Microsoft.HDInsight
 
+### Azure Policy Extension for Arc is not compatible on Kubernetes 1.25 (preview) version
+
+Policy extension for Arc installation will fail on 1.25 clusters with the following error code via gatekeeper controller and policy webhook pods:
```
Comment (nreisch) [inline README.md:276]: consider removing "via gatekeeper controller and policy webhook pods"

File: README.md [hunk 2]
```diff
@@ -278,4 +271,12 @@ This behavior is also seen in resource types from the following RPs:
 - Microsoft.DBforMySQL
 - Microsoft.HDInsight
 
+### Azure Policy Extension for Arc is not compatible on Kubernetes 1.25 (preview) version
+
+Policy extension for Arc installation will fail on 1.25 clusters with the following error code via gatekeeper controller and policy webhook pods: 
+Code: ExtensionOperationFailed
```
Comment (nreisch) [inline README.md:277]: consider adding error message - "err [unable to build kubernetes objects from release manifest: 
 unable to recognize "": no matches for kind "PodSecurityPolicy" in version "policy/v1beta1"]} occurred while doing the operation : "

File: README.md (hunk 1 above)
Comment (nehakulkarni123) [reply README.md]: fixed!

File: README.md (hunk 2 above)
Comment (nehakulkarni123) [reply README.md:277]: fixed!

**Review by nreisch — APPROVED** [review]

**Review by robga — APPROVED** [review]

**Pull request merged**